docs/api-updates.md

```diff
@@ -14,7 +14,7 @@ create a pull request to update Cloudflare docs at the same time.
 
 ## Compatibility documentation in Cloudflare docs
 
-Compatibility flags and dates should be added to the [Cloudflare Workers Compatibility Dates](https://developers.cloudflare.com/workers/platform/compatibility-dates) documentation.
+Compatibility flags and dates should be added to the [Cloudflare Workers Compatibility flags](https://developers.cloudflare.com/workers/configuration/compatibility-flags/) and [dates](https://developers.cloudflare.com/workers/configuration/compatibility-dates/) documentation.
 
 The [cloudflare-docs](https://github.com/cloudflare/cloudflare-docs) repository is hosted on GitHub.
-Compatibility flags and dates are documented in per-feature markdown files under [content/workers/_partials/_platform-compatibility-dates](https://github.com/cloudflare/cloudflare-docs/tree/production/content/workers/_partials/_platform-compatibility-dates).
+Compatibility flags and dates are documented in per-feature markdown files under [src/content/docs/workers/configuration](https://github.com/cloudflare/cloudflare-docs/tree/production/src/content/docs/workers/configuration).
```

Comment:
This should be `src/content/compatibility-flags` and `https://github.com/cloudflare/cloudflare-docs/tree/production/src/content/compatibility-flags` – we need to link the directory that contains all the different files defining compat flags.